### PR TITLE
entry,processor(ticdc): enhancements to mounter

### LINKED ISSUE #4401: Enhancements to mounter
* reduce unnecessary goroutine
* reduce mounter channel size
* reduce lock contention

### PR DESCRIPTION
**Pull request by overvenus**
### What problem does this PR solve?

* reduce unnecessary goroutine
* reduce mounter channel size
* reduce lock contention

Issue Number: close #4401

### Check List 

Tests 

 - Unit test

Code changes

 - Has interface methods change

### Release note 

```release-note
None
```

### CODE REVIEW

File: cdc/entry/mounter.go [hunk 1]
```diff
@@ -148,26 +174,13 @@ func (m *mounterImpl) codecWorker(ctx context.Context, index int) error {
 	}
 }
 
-func (m *mounterImpl) Input() chan<- *model.PolymorphicEvent {
-	return m.rawRowChangedChs[rand.Intn(m.workerNum)]
```
Comment (overvenus) [inline cdc/entry/mounter.go:152]: Note: the global rand has a global lock.

**Review by asddongmen — APPROVED** [review]

File: cdc/entry/mounter.go [hunk 2]
```diff
@@ -39,7 +39,9 @@ import (
 )
 
 const (
-	defaultOutputChanSize = 128000
+	// The buffer size of input channel of each mounter worker.
+	// 16 is larger enough, because a channel exclusively belongs to a worker.
```
Comment (overvenus) [inline cdc/entry/mounter.go:43]:
```suggestion
	// 16 is large enough, because a channel exclusively belongs to a worker.
```

**Review by 3AceShowHand — APPROVED** [review]

**Pull request merged**